### PR TITLE
Resolved type error in progress update 

### PR DESCRIPTION
**Pull request by korakoe**
causes an exception error, the progress bar still updates, and the rest of the launcher remains stable

### CODE REVIEW

**Comment by zenith391** [comment]
I think it would be better if there was a way to keep it and see what actually causes the error and handle it.

**Comment by korakoe** [comment]
@zenith391 I believe I’ve resolved the issue

File: html/index.js [hunk 1]
```diff
@@ -326,7 +326,7 @@ async function downloadMod(version) {
 			downloaded += data.length;
 			requestAnimationFrame(function() {
 				var percent = (downloaded / length) * 100;
-				ipcRenderer.send("update-download", downloaded / length);
+				ipcRenderer.send("update-download", parseint(downloaded / length));
```
Comment (zenith391) [inline html/index.js]: Isn't it `parseInt` and not `parseint` ?

**Review by zenith391 — CHANGES_REQUESTED** [review]

File: html/index.js (hunk 1 above)
Comment (korakoe) [reply html/index.js]: Correct

**Comment by zenith391** [comment]
Thank you

**Pull request merged**